src/uar/mod.rs

```diff
@@ -0,0 +1,290 @@
+mod csv;
```

Comment:
I think this module may want to be split up between all the different relevant modules because I don't think UAR is an audit concept in itself (this whole repo is for that). Thoughts?